packages/agent/src/hooks/useDidCommPresentationActions.ts

```diff
@@ -155,8 +155,7 @@ export function useDidCommPresentationActions(proofExchangeId: string) {
               return {
                 credential: credentialForDisplay,
                 disclosed: {
-                  // TODO: we don't show this yet on anoncreds screen, but we should add it
-                  attributes: {},
+                  attributes: credentialForDisplay.attributes,
```

Comment:
This should only be the disclosed attributes?

Reply:
fixed